src/NativeShell/plugin.xml

```diff
@@ -13,27 +13,24 @@
   <platform name="android">
     <config-file target="res/xml/config.xml" parent="/*">
       <feature name="NativeShell">
-        <param name="android-package" value="org.jellyfin.cordova.NativeShell"/>
+        <param name="android-package" value="org.jellyfin.mobile.NativeShell"/>
       </feature>
     </config-file>
+    <config-file target="AndroidManifest.xml" parent="/manifest">
+        <uses-permission android:name="android.permission.WAKE_LOCK"/>
+    </config-file>
     <config-file target="AndroidManifest.xml" parent="/manifest/application">
-      <service android:name="com.mb.android.media.RemotePlayerService" />
+      <service android:name="org.jellyfin.mobile.RemotePlayerService" />
     </config-file>
     <source-file src="libs/apiclient.jar" target-dir="libs" />
     <source-file src="src/build-extras.gradle" target-dir="." />
-    <source-file src="src/cordova/NativeShell.java" target-dir="src/org/jellyfin/cordova" />
-    <source-file src="src/api/ApiClientBridge.java" target-dir="src/com/mb/android/api" />
-    <source-file src="src/logging/AppLogger.java" target-dir="src/com/mb/android/logging" />
-    <source-file src="src/logging/LogbackLogger.java" target-dir="src/com/mb/android/logging" />
-    <source-file src="src/logging/LogFileFilter.java" target-dir="src/com/mb/android/logging" />
-    <source-file src="src/media/RemotePlayerService.java" target-dir="src/com/mb/android/media" />
+    <source-file src="src/NativeShell.java" target-dir="src/org/jellyfin/mobile" />
+    <source-file src="src/AppLogger.java" target-dir="src/org/jellyfin/mobile" />
+    <source-file src="src/LogbackLogger.java" target-dir="src/org/jellyfin/mobile" />
+    <source-file src="src/RemotePlayerService.java" target-dir="src/org/jellyfin/mobile" />
+    <source-file src="src/CustomWebViewClient.java" target-dir="src/org/jellyfin/mobile" />
     <framework src="com.android.support:support-v4:23.3.0" />
-    <framework src="com.android.support:appcompat-v7:23.3.0" />
-    <framework src="com.android.support:mediarouter-v7:23.3.0" />
```

Comment:
why these two are no longer needed?

Reply:
They were used in the chromecast plugin which is now set up in another repository.